conformance/utils/kubernetes/helpers.go

```diff
@@ -89,6 +95,38 @@ func GWCMustBeAccepted(t *testing.T, c client.Client, timeoutConfig config.Timeo
 	return controllerName
 }
 
+func HTTPRouteConditionsHaveLatestObservedGeneration(r *v1beta1.HTTPRoute) bool {
+	for _, p := range r.Status.Parents {
+		if !ConditionsHaveLatestObservedGeneration(r, p.Conditions) {
+			return false
+		}
+	}
+	return true
+}
+
+func GatewayClassConditionsHaveLatestObservedGeneration(gc *v1beta1.GatewayClass) bool {
+	if !ConditionsHaveLatestObservedGeneration(gc, gc.Status.Conditions) {
```

Comment:
this and below can be simplified to `return ConditionsHaveLatestObservedGeneration(gc, gc.Status.Conditions)`